crates/sui-pg-db/src/lib.rs

```diff
@@ -126,39 +123,27 @@ impl Db {
         Ok(())
     }
 
-    /// Run migrations on the database. Use the `migrations` parameter to pass in the migrations
-    /// that are specific to the indexer being run. Migrations that the indexer framework needs
-    /// will be added automatically.
-    ///
-    /// Use Diesel's `embed_migrations!` macro to generate the `migrations` parameter for your
-    /// indexer.
-    pub(crate) async fn run_migrations(
+    /// Run migrations on the database. Use Diesel's `embed_migrations!` macro to generate the
+    /// `migrations` parameter for your indexer.
+    pub async fn run_migrations<S: MigrationSource<Pg> + Send + Sync + 'static>(
```

Comment:
so unlike before, now we don't apply the migrations that the indexer framework needs - it's up to the user then to provide the necessary migrations?

Reply:
In this package we don't add any migrations, but the `sui-indexer-alt-framework` crate still does that. This change makes `run_migrations` more general, so that the framework is able to provide the combined migrations.